manifests/charts/ambient/values.yaml

```diff
@@ -0,0 +1,32 @@
+# Copied from `helm-profiles/ambient.yaml`
+
+global:
```

Comment:
Helm noob question: does this end up as `global.variant` in the chart or does it flatten it to `variant`?

Or is it that each chart gets
* global -> global
* <chart name> -> flattened
?

Reply:
`global.xxx` is never scoped/rescoped - it is always `global.xx` in all charts + all subcharts.

Reply:
And if you redefine the global in a subchart:

> If a subchart declares a global variable, that global will be passed downward (to the subchart's subcharts), but not upward to the parent chart. There is no way for a subchart to influence the values of the parent chart.

> Also, global variables of parent charts take precedence over the global variables from subcharts.